src/lib/keypom-utils.ts

```diff
@@ -281,12 +283,26 @@ export const ftTransferCall = ({
 // TODO: Document this
 export const nftTransferCall = async ({
     account,
+    wallet,
     contractId,
     receiverId,
     tokenIds,
     msg,
     returnTransactions = false,
 }: NFTTransferCallParams): Promise<Array<void | FinalExecutionOutcome[]> | Transaction[]> => {
+
+    const { getAccount } = getEnv();
+
+	account = getAccount({ account, wallet })
+
+    if (!account) {
```

Comment:
`getAccount` already throws if there's no valid account so this is redundant

Reply:
https://media.tenor.com/0zQRsHJRFQEAAAAC/hulk-hogan-nodding.gif